atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/integrationInput/IntegrationTestTableFactory.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.table.description.integrationInput;
+
+import java.util.List;
+
+import javax.annotation.Generated;
+
+import com.google.common.base.Function;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Multimap;
+import com.palantir.atlasdb.keyvalue.api.Namespace;
+import com.palantir.atlasdb.table.generation.Triggers;
+import com.palantir.atlasdb.transaction.api.Transaction;
+
+@Generated("com.palantir.atlasdb.table.description.render.TableFactoryRenderer")
+public final class IntegrationTestTableFactory {
```

Comment:
I thought the idea was to not generate the code and make the test task depend on generateSchemas.